packages/datadog_session_replay/lib/datadog_session_replay.dart

```diff
@@ -7,7 +7,7 @@ import 'package:datadog_flutter_plugin/datadog_flutter_plugin.dart';
 import 'src/datadog_session_replay_plugin.dart';
 
 export 'src/datadog_session_replay.dart' show DatadogSessionReplay;
-export 'src/widgets.dart' show SessionReplayCapture;
+export 'src/widgets.dart' show SessionReplayCapture, SessionReplayPrivacy;
 
 /// Privacy levels for text and input masking in Session Replay
```

Comment:
tiny nit: align with others
```suggestion
/// Available privacy levels for text and input masking in Session Replay.
```